Runtime/Scripts/LyumaAv3Runtime.cs

```diff
@@ -1920,6 +1941,7 @@ public void SetupCloneCaches()
 		}
 
 		void Update() {
+			animator ??= GetOrAddComponent<Animator>(this.gameObject);
```

Comment:
??= is C# 9 feature, which is 2019 incompatible.
in addition, ?? doesn't handle destroyed objects, only actual null.

```suggestion
			if (animator == null) animator = GetOrAddComponent<Animator>(this.gameObject);
```